### PR TITLE
[T4A3][T11-A2]Jordan May

### PR DESCRIPTION
**Pull request by jjamay**
Ready for review

### CODE REVIEW

**Comment by jjamay** [comment]
I understand the unit test signature standards.

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -1,59 +1,35 @@
...
     /**
      * Validates given address.
      *
      * @throws IllegalValueException if given address string is invalid.
      */
     public Address(String address, boolean isPrivate) throws IllegalValueException {
-        String trimmedAddress = address.trim();
-        this.isPrivate = isPrivate;
-        if (!isValidAddress(trimmedAddress)) {
+        super(address, isPrivate);
```
Comment (nirandiw) [inline src/seedu/addressbook/data/person/Address.java:22]: You can push the whole constructor to the super class. Need to pass the validation regex and the error message.

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
```
Comment (nirandiw) [inline src/seedu/addressbook/data/person/Contact.java:4]: Header comments are missing. Make sure NOT to mention the child classes in the header comments of a super class.

**Review by nirandiw — COMMENTED** [review]
@jjamay Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**